src/VivViewerOverview.js

```diff
@@ -0,0 +1,143 @@
+import { OrthographicView } from '@deck.gl/core';
+import { OverviewLayer } from './layers';
+
+export default class VivViewerOverview {
+  constructor({
+    viewWidth,
+    viewHeight,
+    loader,
+    offset = 25,
+    overviewLocation = 'bottom-right',
+    id = 'overview',
+    boundingBoxColor = [255, 0, 0],
+    boundingBoxOutlineWidth = 50,
+    viewportOutlineColor = [255, 192, 204],
+    viewportOutlineWidth = 400,
+    overviewScale = 1
+  }) {
+    const { numLevels } = loader;
+    const rasterSize = loader.getRasterSize({
+      z: 0
+    });
+    this.offset = offset;
+    this.viewWidth = viewWidth;
+    this.viewHeight = viewHeight;
+    this.id = id;
+    this._numLevels = numLevels;
+    // These are the pixel-space height and width
+    this.width = viewWidth * overviewScale;
+    this.height = this.width * (rasterSize.height / rasterSize.width);
```

Comment:
I think this is what you had in mind @manzt?

Reply:
yeah exactly!!